src/services/findHouse.js

```diff
@@ -0,0 +1,41 @@
+export default function findHouse(quizResults, questionArray) {
+  const houseQuizAnswers = Object.entries(quizResults)
+
+  if (!houseQuizAnswers.length)
+    return 'Something is wrong with your data, Muggle!'
+
+  const listOfHousesAccordingToAnswers = houseQuizAnswers.flatMap(
+    answerArray => {
+      const associatedQuestion = questionArray.find(
+        el => el.question === answerArray[0]
+      )
+      const possibleAnswers = Object.entries(associatedQuestion.answers) // [[house, answer], ...]
+
+      const houses = possibleAnswers
```

Comment:
you could directly return the array as the additional variable doesn't add much to the readability (and it is clear from the function name that it will return houses)